src/components/content-box/_content-box.scss

```diff
@@ -34,7 +34,7 @@ $include-html: false !default;
 
     &--spaced {
       .mint-content-box__content, .mint-content-box__footer {
-        @media (min-width: $breakpointMedium) {
+        @include mintBreakpoint(medium-up) {
```

Comment:
Ideally, I would call variables with dashes, to have it consistent with class names, but this is a different story.